lib/resources/collection/index.js

```diff
@@ -86,7 +86,7 @@ Collection.prototype.validate = function (body, create) {
         errors[key] = 'is required';
       }
     } else if(type === 'boolean') {
-      body[key] = false;
+      body[key] = val;
```

Comment:
Leaving this here although we discussed on gitter. This else block is only going to be executed when val is undefined, otherwise the top if block will catch it. 

There was no bug here, the previous version of this file was correct by setting the boolean property to false when an undefined/missing value is passed in.